--- common.inc.orig	2005-12-02 12:30:59.000000000 +0200
+++ common.inc	2005-12-25 22:43:54.055438336 +0200
@@ -55,11 +55,12 @@
  * Retrieve output to be displayed in the head tag of the HTML page.
  */
 function drupal_get_html_head() {
-  global $base_url;
+  global $base_url, $locale;
 
   $output = "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\" />\n";
   $output .= "<base href=\"$base_url/\" />\n";
-  $output .= theme('stylesheet_import', 'misc/drupal.css');
+  $ltr = in_array($locale, array('ar', 'fa', 'he', 'ur')) ? '' : '-ltr';
+  $output .= theme('stylesheet_import', 'misc/drupal' . $ltr . '.css');
 
   return $output . drupal_set_html_head();
 }
