Index: /home/workspace/fh47live/modules/ecommerce/product/product.module
===================================================================
--- /home/workspace/fh47live/modules/ecommerce/product/product.module	(revision 220)
+++ /home/workspace/fh47live/modules/ecommerce/product/product.module	(working copy)
@@ -1138,7 +1138,8 @@
     $now = time();
     $variables['%site'] = variable_get('site_name', 'drupal');
     $variables['%uri'] = url('', NULL, NULL, TRUE);
-    $from = variable_get('site_mail', ini_get('sendmail_from'));
+    // Using store_mail instead of site_mail, double fallback
+    $from = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
 
     foreach($payments as $p) {
       //Build the variable list that can be expanded in the email msg.
Index: /home/workspace/fh47live/modules/ecommerce/shipping/shipping.module
===================================================================
--- /home/workspace/fh47live/modules/ecommerce/shipping/shipping.module	(revision 220)
+++ /home/workspace/fh47live/modules/ecommerce/shipping/shipping.module	(working copy)
@@ -685,7 +685,8 @@
 function shipping_shipping_notification() {
   $t = store_transaction_load(arg(4));
   $variables  = store_email_get_transaction_variables($t);
-  $from = variable_get("site_mail", ini_get("sendmail_from"));
+  // Using store_mail instead of site_mail, double fallback
+  $from = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
 
   if ($t->mail && valid_email_address($t->mail)) {
     $to = $t->mail;
Index: /home/workspace/fh47live/modules/ecommerce/store/store.module
===================================================================
--- /home/workspace/fh47live/modules/ecommerce/store/store.module	(revision 220)
+++ /home/workspace/fh47live/modules/ecommerce/store/store.module	(working copy)
@@ -78,8 +78,8 @@
       $output .= t("Want more reports? Visit %base_url/admin/store\n", array('%base_url' => $base_url));
 
       $subject    = t('%site-name transaction summary', array('%site-name' => variable_get('site_name', 'Drupal')));
-      $to         = variable_get("site_mail", ini_get("sendmail_from"));
-      $from       = variable_get("site_mail", ini_get("sendmail_from"));
+      $to         = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
+      $from       = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
       $headers    = "From: $from\nReply-to: $from\nX-Mailer: PHP\nReturn-path: $from\nErrors-to: $from";
 
       user_mail($to, $subject, $output, $headers);
@@ -206,8 +206,8 @@
     '#options' => $options,
     '#description' => t('There are several advantages in having customers create accounts. When they shop, the items in their cart will be remembered from visit to visit, and they can store their shipping and billing addresses in an address book at this site.')
   );
-
-  $site_mail = variable_get('site_mail', ini_get('sendmail_from'));
+  // Using store_mail instead of site_mail, double fallback
+  $site_mail = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
   if (!$site_mail) {
     $site_mail = 'no e-mail address found';
   }
@@ -1125,7 +1125,7 @@
 
   $t = store_transaction_load($txnid);
   $variables  = store_email_get_transaction_variables($t);
-  $from = variable_get("site_mail", ini_get("sendmail_from"));
+  $from = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
 
   if ($t->mail && valid_email_address($t->mail)) {
     $to = $t->mail;
@@ -1156,7 +1156,7 @@
 function store_send_error_email($txnid, $preview = null) {
   $t = store_transaction_load($txnid);
   $variables  = store_email_get_transaction_variables($t);
-  $from = variable_get("site_mail", ini_get("sendmail_from"));
+  $from = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
 
   if ($t->mail && valid_email_address($t->mail)) {
     $to = $t->mail;
@@ -1187,7 +1187,7 @@
 function store_transaction_ask_customer() {
   $t = store_transaction_load(arg(4));
   $variables  = store_email_get_transaction_variables($t);
-  $from = variable_get("site_mail", ini_get("sendmail_from"));
+  $from = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
 
   if ($t->mail && valid_email_address($t->mail)) {
     $to = $t->mail;
@@ -1242,7 +1242,7 @@
 function store_transaction_cancel() {
   $t = store_transaction_load(arg(4));
   $variables  = store_email_get_transaction_variables($t);
-  $from = variable_get("site_mail", ini_get("sendmail_from"));
+  $from = variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from')));
 
   if ($t->mail && valid_email_address($t->mail)) {
     $to = $t->mail;
@@ -2491,7 +2491,7 @@
     '%shipping_method' => $shipping_method,
     '%user_data' => $shipping_to . $billing_to . $shipping_method,
     '%items' => $items,
-    '%email' => variable_get('site_mail', ini_get('sendmail_from')),
+    '%email' => variable_get('store_mail', variable_get('site_mail', ini_get('sendmail_from'))),
     '%site' => variable_get('site_name', 'drupal'),
     '%uri' => $base_url,
     '%uri_brief' => substr($base_url, strlen("http://")),
